dialogue/TownNPCs/Olive/InitialInteractions/Pistachio/Pistachio_Olive_First3.json

```diff
@@ -11,11 +11,17 @@
       "At the very least...\nif you do make it back, let me listen to your adventures.",
       "That way I can make sure you're not doing anything too dangerous..!"
     ],
+    "pl": [
+      "P-Poważnie..?\nWygląda na to, że nie mogę cię powstrzymać...",
+      "Najmniej...\nJeśli uda ci się wrócić, opowiedz mi o twoich przygodach.",
```

Comment:
Use "Przynajmniej" instead of "Najmniej". Also I think "Jeśli wrócisz" will be better than "Jeśli uda ci się wrócić", but both are correct